src/coreclr/vm/ClrEtwAll.man

```diff
@@ -1775,6 +1775,30 @@
                         </UserData>
                     </template>
 
+                    <template tid="WaitStart_V1">
+                        <data name="ClrInstanceID" inType="win:UInt16" />
+                        <data name="LockID" inType="win:Pointer" />
+                        <data name="AssociatedObjectID" inType="win:Pointer" />
+                        <UserData>
```

Comment:
It could be interesting to add a `ThreadID` property. It would also help to have a `IsThreadPoolThread` but I don't think there is a concept of thread pool at that level

Reply:
I think thread id is added automatically to every emitted event (the same way as timestamp)